Makefile

```diff
@@ -67,6 +67,7 @@ $(AE_DEIDENTIFIED_DATA) &: $(LONDON_POSTCODES)
 $(SYNTH_OUTPUTS_CSV) : \
 synth-output/%/synthetic_data_1.csv : run-inputs/%.json $(AE_DEIDENTIFIED_DATA)
 	mkdir -p $$(dirname $@) && \
+	cp $< $$(dirname $@) && \
```

Comment:
Is this needed later in the pipeline. In case of big files we might want to avoid copying since the file is already available?

Reply:
That is a good point, but do you think the input file will be large? For reproducibility, I thought it would be good to always keep the input file inside the output directory (I was doing it manually, so this is why I added it here).

Reply:
However, if you think it is not necessary, I am also happy to remove this line.

Reply:
It is possible QUIPP will be used with big data sets in the future. Also, it is probably better to have the input file in one unique location.

Reply:
OK, we discussed this, and since `cp $< $$(dirname $@)` copies the input (json) file (and not the input dataset), we agreed to have this line in the Makefile for reproducibility.